Example/Sources/Root/ClientRewardProvider.swift

```diff
@@ -54,6 +55,11 @@ extension ClientRewardProvider: LucraRewardProvider {
             UIApplication.shared.open(url)
         }
     }
+    
+    
+    func viewRewards() {
+        
+    }
```

Comment:
@wellison-lucra Is this function empty by design?

Reply:
For this implementation it's just for protocol conformance, in our internal sample we display a view that is not hooked up for this version of the sample app.